index.js

```diff
@@ -1,32 +1,45 @@
-const client = require('./lib/statsd');
+/*
+ * Initialize a hot-shots client.
+ */
 
-const isNotEmpty = (obj) => obj && Object.getOwnPropertyNames(obj).length;
+const os = require('os');
 
-function keyValuesToString(keyValues) {
-  return Object.keys(keyValues)
-    .reduce((r, k) => r.concat(`${k}=${keyValues[k]}`), [])
-    .join(',');
+const statsdClient = require('hot-shots');
+const strigoLogger = require('strigo-node-logger');
+
+const STATSD_HOST = '127.0.0.1';
+const STATSD_PORT = 8125;
+const DEFAULT_ENV = os.hostname();
+const CONTAINER_HOSTNAME = process.env.HOSTNAME || 'null';
+
+const log = strigoLogger.setupNodeLogger({ env: DEFAULT_ENV, level: 'info' });
+
+function setupStatsdClient({ svc, env = DEFAULT_ENV, host = STATSD_HOST, port = STATSD_PORT }) {
+  if (!svc) {
+    throw new Error('Must provide a service name');
+  }
+
+  log.info(`Initializing statsd client for service ${svc} @ ${STATSD_HOST}:${STATSD_PORT}...`);
+  const client = new statsdClient.StatsD(
+    {
+      host,
+      port,
+      prefix: `${svc}_`,
+      globalTags: {
+        env,
```

Comment:
will the env tag will be in "env: ____" format?

Reply:
No. `env=env`

Reply:
ok, yea that good